examples/ecs/entity_disabling.rs

```diff
@@ -0,0 +1,155 @@
+//! Disabling entities is a  powerful feature that allows you to hide entities from the ECS without deleting them.
+//!
+//! This can be useful for implementing features like "sleeping" objects that are offscreen
+//! or managing networked entities.
+//!
+//! Note that you should *not* use this feature to simply make entities invisible!
```

Comment:
It's a little odd that this comment says not to use this feature to make entities invisible, yet the example makes entities invisible! Still I guess its sorta the easiest way to showcase the example

Reply:
Yeah, that's part of why the name-listing system exists :) It will *also* make things invisible of course!